src/NuGet.Core/NuGet.Commands/RestoreCommand/RestoreSummary.cs

```diff
@@ -159,6 +156,32 @@ public static void Log(ILogger logger, IEnumerable<RestoreSummary> restoreSummar
             }
         }
 
+        private static void LogToConsole(
+            RestoreSummary restoreSummary,
+            LogLevel logLevel,
+            string logHeading,
+            LogString log,
+            LogString logSeparator)
```

Comment:
this is really confusing to me...

Reply:
+1. I prefer passing functions as Actions just so it's clear that it's a function and not a data structure

Reply:
reading the code, might help :P